cleaners/opera.xml

```diff
@@ -78,8 +86,8 @@ Notes about Opera
     <!-- below are entries valid before Opera 15 -->
     <!-- example: C:\Documents and Settings\user\Application Data\Opera\Opera\cookies.txt -->
     <!-- example: C:\Documents and Settings\user\Application Data\Opera\Opera 11.00 beta\cookies.txt -->
-    <action command="delete" search="glob" path="$appdata\Opera\Opera*\cookies4.dat"/>
-    <action command="delete" search="glob" path="$appdata\Opera\Opera*\profile\cookies4.dat"/>
+    <action command="delete" search="file" path="$appdata\Opera\Opera*\cookies4.dat"/>
+    <action command="delete" search="file" path="$appdata\Opera\Opera*\profile\cookies4.dat"/>
```

Comment:
When you put a wildcard in `path`, then set `search="glob"` 

I don't understand the reason for this change.

Reply:
Yeah, was BS! ^^